ui/app/AppLayouts/Onboarding2/OnboardingFlow.qml

```diff
@@ -64,6 +67,11 @@ OnboardingStackView {
 
     signal finished(int flow)
 
+    // Thirdparty services
+    property bool privacyModeFeatureEnabled
```

Comment:
Is this a wip one, right? Maybe a comment to understand that this will be removed once we remove the feature flag would help!

Reply:
Yeah, I meant if this is just a temporal flag that will be removed when the complete feature is ready